pilot/docker/wait-ready.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+url="${1:-localhost:15021/healthz/ready}"
+
+echo "Waiting for proxy to become ready..."
+while true; do
+  responseCode=$(curl -s --connect-timeout 1 -o /dev/null -w "%{http_code}" "${url}")
+  if [ "$responseCode" == "200" ]; then
+    echo "Proxy is ready!"
+    exit 0
+  fi
+  echo "Proxy not ready (response code: ${responseCode})..."
```

Comment:
add timestamp

Reply:
I've replaced the script with `pilot-agent wait`, which uses regular istio logging, so the timestamps are now also printed.